src/lib.rs

```diff
@@ -1128,14 +1128,14 @@ impl UInt {
     ///
     /// ```
     /// # use js_int::UInt;
-    /// assert_eq!(UInt::from(5u32).overflow_mul(UInt::from(2u32)), (UInt::from(10u32), false));
+    /// assert_eq!(UInt::from(5u32).overflowing_mul(UInt::from(2u32)), (UInt::from(10u32), false));
     /// assert_eq!(
-    ///     UInt::max_value().overflow_mul(UInt::from(2u32)),
+    ///     UInt::max_value().overflowing_mul(UInt::from(2u32)),
```

Comment:
`UInt::max_value` has been deprecated in the meantime since this PR was opened. The other tests have already changed to using `UInt::MAX`, can you do the same for the new methods?